lib/elixir/src/elixir_module.erl

```diff
@@ -35,6 +35,11 @@ delete_doc(#{module := Module}, _, _, _, _, _) ->
   ets:delete(data_table(Module), doc),
   ok.
 
+delete_impl(#{module := Module}, _, _, _, _, _) ->
+  ets:delete(data_table(Module), impl),
+  % TODO: What to do here?
```

Comment:
Nothing. :) This is during bootstrap, so we don't care. If there are warnings, the modules will be recompiled after bootstrap, so we get them anyway.